File: CFLServerData/CFL_Data/Lieu.cs
using MSTD.ShBase;

namespace CFL_1.CFL_Data
{
    /// <summary>
    /// Lieu represente tout lieu (contient Coordonnees et Contacts)
    /// </summary>
    public class Lieu : Base
    {
        public Coordonnees adresse { get; set; } = new Coordonnees();
        public Contacts contacts { get; set; } = new Contacts();
    }
}

File: src/cflserver/cfl_data/coordonnees/lieu.py
from mongoengine import *

from src.cflserver.cfl_data.coordonnees.adresse import Adresse
from src.cflserver.cfl_data.coordonnees.contacts import Contacts

class Lieu(Document):
    nom = StringField()
    adresse = ReferenceField(Adresse, default = None)
    contacts = ReferenceField(Contacts, default = None)
